win32ss/user/user32/windows/cursoricon.c

```diff
@@ -221,7 +450,7 @@ static int DIB_GetBitmapInfo( const BITMAPINFOHEADER *header, LONG *width,
     }
     if (memcmp(&header->biSize, png_sig_pattern, sizeof(png_sig_pattern)) == 0)
     {
-        ERR("Cannot yet display PNG icons\n");
+        TRACE("We have a PNG icon\n");
         /* for PNG format details see https://en.wikipedia.org/wiki/PNG */
```

Comment:
Shouldn't you fill the width, height, bpp and compr here ?

Reply:
Nope. `DIB_GetBitmapInfo` assumes bitmap format. If it is not a bitmap, its info is useless.